Gopkg.lock

```diff
@@ -397,6 +397,16 @@
   revision = "5923b6288fe8ce9581936ee97c2bf9cf9c02c2f4"
   version = "v0.22.0-rc2"
 
+[[projects]]
+  name = "github.com/tendermint/tmlibs"
```

Comment:
Why is `tmlibs` included? It has since been merged into Tendermint, and we should import `tmlibs` as a Tendermint subpackage in all cases.